os/board/rtl8730e/src/component/bluetooth/os/tizenrt/osif_tizenrt.c

```diff
@@ -115,7 +115,6 @@ bool osif_task_create(void **pp_handle, const char *p_name, void (*p_routine)(vo
 	task_info[0] = itoa((int) p_routine, routine_addr, 16);
 	task_info[1] = itoa((int) p_param, param_addr, 16);
 	task_info[2] = NULL;
-	stack_size = stack_size * sizeof(uint32_t);
```

Comment:
It looks good change because it makes mistakes.
But this change makes all stack sizes of threads created by this function 4 times smaller than before.
Did you verify all operations related to this change? Please test all related operations.